pkgs/misc/vim-plugins/deprecated.json

```diff
@@ -1,6 +1,6 @@
 {
     "compe-tmux": {
-        "date": "2021-12-07",
+        "date": "2021-12-10",
```

Comment:
i think the update script is doing something weird, can you drop this commit?

cc @teto if you want to take a look at the update script

Reply:
Done.